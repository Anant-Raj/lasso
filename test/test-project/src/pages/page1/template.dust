{@optimizer-page name="page1" packagePath=packagePath /}

<html>
    <head>
        {@optimizer-head /}
    </head>
    <body>
        {@optimizer-body /}
    </body>
</html>