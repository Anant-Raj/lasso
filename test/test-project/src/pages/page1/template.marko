<optimizer-page name="page1" package-path="./browser.json"/>
<optimizer-resource path="./favicon.ico" var="favicon"/>

<html>
    <head>
        <optimizer-head/>
        <link rel="shortcut icon" href="${favicon.url}">
    </head>
    <body>
        <optimizer-body/>
    </body>
</html>