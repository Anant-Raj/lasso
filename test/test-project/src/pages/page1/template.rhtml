<optimizer-page name="page1" package-path="./optimizer.json"/>

<html>
    <head>
        <optimizer-head/>
    </head>
    <body>
        <optimizer-body/>
    </body>
</html>