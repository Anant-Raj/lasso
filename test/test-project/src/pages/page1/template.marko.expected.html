<html><head><link rel="stylesheet" href="/static/page1-8b866529.css"><link rel="shortcut icon" href="/static/favicon-c3deb101.ico"></head><body><img src="/static/marko-logo-314d77d4.png" width="240" height="200"><img src="/static/marko-logo-314d77d4.png" width="100" height="100"><script src="/static/page1-1097e0f6.js"></script></body></html>